man/src/network/create.md

```diff
@@ -134,3 +134,44 @@ $ docker network create -d overlay \
   --opt encrypted=true \
   my-ingress-network
 ```
+
+### Run services on predefined networks
+
+You can now create services on the predefined docker networks `bridge` and `host`.
+
+```bash
+$ docker service create --name my-service \
+  --network host \
+  --replicas 2 \
+  busybox top
+```
+
+### Swarm networks with local scope drivers
+
+You can now create a swarm network with local scope network drivers. You do so
+by promoting the network scope to `swarm` during the creation of the network. 
+You will then be able to use this network when creating services. 
+
+```bash
+$ docker network create -d bridge \
+  --scope swarm \
+  --attachable \
+  swarm-network
+```
+
+For network drivers which provide connectivity across hosts (ex. macvlan), if
+node specific configurations are needed in orther to plumb the network on each
+host, you will supply that configuration via a configuration only network.
+When you create the swarm scoped network, you will then specify the name of the 
+network which contains the configuration.
+
+
+```bash
+node1$ docker network create --config-only --subnet 192.168.100.0/24 --gateway 192.168.100.115 mv-config
```

Comment:
We see the first class network configuration options are part of two categories: `network specific` (`subnet`, `mtu`,...) and `operator specific` (`internal`, `attachable`, `ingress`,...)
`--scope` is operator specific.

Config-only networks are meant to carry only network specific configurations. Operator specific configurations, if applied, are discarded either explicitly or implicitly.

Given the config network is never plumbed, never passed down to the network drivers, the `driver` option is discarded.

It is perfectly fine to create a 
`docker network create --config-only --mtu=1460 conf-nw` 
and apply it to `macvlan`, `bridge` or `ipvlan` networks.